PusherClient/EventEmitter.cs

```diff
@@ -16,7 +16,9 @@ public void Bind(string eventName, Action<dynamic> listener)
         {
             if(_eventListeners.ContainsKey(eventName))
             {
-                _eventListeners[eventName].Add(listener);
+                // To avoid triggering the same bind multiple times

```

Comment:
We need to support more than one bind per event name because multiple pieces of application logic may want to know about the same event e.g. things like connection events, which also uses the EventEmitter.